File: basic/cs205_c_cpp_2020s/lab13/rusrc/lab13_01.rs
pub mod lib;

mod lab13_01 {
    use crate::lib::c_stereo_shape;
    use crate::lib::c_stereo_shape::CStereoShape;
    use crate::lib::shape::shape::Shape;

    pub(crate) fn main() {
        let css1 = CStereoShape::default();
        println!("{}", css1.get_area());
        println!("{}", css1.get_volumn());
        css1.show();
        println!("value is {}", CStereoShape::value());
    }
}

fn main() {
    lab13_01::main();
}

File: basic/cs205_c_cpp_2020s/lab13/exercise/lab13_01.cpp
// SPDX-License-Identifier: AGPL-3.0-or-later
/*  CS205_C_CPP
    Copyright (C) 2020-2022 nanoseeds
*/
/**
 * @Github: https://github.com/Certseeds
 * @Author: nanoseeds
 * @Date: 2020-05-17 10:34:34
 * @LastEditors  : nanoseeds
 */
#include "c_stereoShape_lab13_01.h"
#include <iostream>

int main() {
    CStereoShape css1{};
    std::cout << css1.GetArea() << "\n";
    std::cout << css1.GetVolume() << "\n";
    css1.Show();
    CStereoShape::GetNumOfObject();
    return 0;
}